neurolib/models/bold/timeIntegration.py

```diff
@@ -85,6 +105,7 @@ def integrateBOLD_numba(BOLD, X, Q, F, V, Z, dt, N, rho, alpha, V0, k1, k2, k3,
             V[j] = V[j] + dt / Tau[j] * (F[j] - V[j] ** (1 / alpha))
             F[j] = F[j] + dt * X[j]
 
-            BOLD[j, i] = V0 * (k1 * (1 - Q[j]) + k2 * (1 - Q[j] / V[j]) + k3 * (1 - V[j]))
+            F[j] = max(F[j], EPS)
```

Comment:
`F` represents blood inflow to the windkessel. I think it is supposed to be bounded between 0 and 1, but it overflows sometimes. Was going negative (or very small) in some weird dynamical scenarios. Had to do a lot of debugging of the equations to find out what exacly overflows but it boils down to the integration of `F`. I suppose that a better integration scheme would avoid this problem. For now this is the easiest fix. 

I have tested whether this affects BOLD outputs and it barely does (the constant is now `1E-120` instead of the `1E-3` I've added to line 106 before, which had an effect on the timeseries, which shouldn't happen...).